Python/pywarpx/picmi.py

```diff
@@ -432,6 +383,34 @@ def setup_parse_momentum_functions(self, species):
                 expression = f'{self.directed_velocity[idir]}'
             species.__setattr__(f'momentum_function_u{sdir}(x,y,z)', f'({expression})/{constants.c}')
 
+
+class UniformDistribution(DensityDistributionBase, picmistandard.PICMI_UniformDistribution):
```

Comment:
For the documentation to work properly (i.e. for it to include the doc from the picmistandard class), the picmistandard class should be first in the list of base classes. You can check this by locally generating the documentation.

Reply:
That is very interesting, thanks for letting me know! I'll swap the order around.